src/evaluation/baselines.py

```diff
@@ -42,19 +42,24 @@ def get_action_list(self):
         action_list = [(method, getattr(MyActions, method).__doc__) for method in action_list]
         return action_list
 
-    def get_encoded_action_list(self):
+    def get_encoded_input_prompt(self, input: Input):
         actions = self.get_action_list()
         # encode the actions as a string
         actions = '\n\n'.join([f"{action[0]}: {action[1]}" for action in actions])
-        return f"""Given a web-based task, we'd like to solve it by executing actions. Below is a list of actions that 
-        can be performed on a HTML page. \n\n{actions}"""
+        return f"""
+        Given a web-based task, we'd like to solve it by executing actions. Below is a list of actions that
+        can be performed on a HTML page. \n\n{actions} \n\n
+
+        Now, given the following input `{input}`, decide what set of actions need to be executed.
+        """
+
 
 
 class NewBaseline(Baseline):
 
     def solve_task(self, input: Input, **kwargs):
         # list of ations that can be performed on a HTML page
-        encoded_actions_prompt = self.get_encoded_action_list()
+        encoded_actions_prompt = self.get_encoded_input_prompt()
```

Comment:
does this work? I feel like we need to pass the input param here since get_encded_input_prompt uses input in its return statement. i know this is untested

Reply:
Ya this function expects an input.

Reply:
`get_encoded_input_prompt(input)`